packages/vue-email/src/components/e-heading.vue

```diff
@@ -1,46 +1,66 @@
 <template>
   <component
-    v-bind="{
-      ...$props,
-      dataId: '__vue-email-heading',
-    }"
     :is="as"
-    :style="
-      styleToString({
-        ...withMargin({
-          m,
-          mx,
-          my,
-          mt,
-          mr,
-          mb,
-          ml,
-        }),
-        ...style,
-      })
-    "
+    data-id="__vue-email-heading"
+    :style="{
+      ...withMargin({
+        m,
+        mx,
+        my,
+        mt,
+        mr,
+        mb,
+        ml,
```

Comment:
Similar to the previous example, passing the `style` prop provides users with the opportunity to add custom styles when they choose not to utilize Tailwind classes. What do you think about this?